src/apps/components/itemSheetsHeader/ItemSheetHeader.svelte

```diff
@@ -1,69 +1,118 @@
 <script>
-    import { getContext } from "svelte";
+  import { getContext } from "svelte";
 
-    import editDocumentImage from "../../handlers/editDocumentImage";
-    import updateDocumentDataFromField from "../../utils/updateDocumentDataFromField";
+  import editDocumentImage from "../../handlers/editDocumentImage";
+  import updateDocumentDataFromField from "../../utils/updateDocumentDataFromField";
 
-    const item = getContext("item");
+  const item = getContext("item");
+  const hasType = ["maneuver", "feature"];
+  const appId = getContext("appId");
 </script>
 
 <header class="sheet-header">
-    <!-- svelte-ignore a11y-click-events-have-key-events -->
-    <img
-        class="item-image"
-        src={$item.img}
-        alt={$item.name}
-        title={$item.name}
-        on:click={() => editDocumentImage($item)}
-    />
+  <!-- svelte-ignore a11y-click-events-have-key-events -->
+  <img
+    class="item-image"
+    src={$item.img}
+    alt={$item.name}
+    title={$item.name}
+    on:click={() => editDocumentImage($item)}
+  />
 
+  <div>
     <input
-        type="text"
-        name="name"
-        value={$item.name}
-        class="item-name"
-        placeholder="Name"
-        on:change={({ target }) =>
-            updateDocumentDataFromField($item, target.name, target.value)}
+      type="text"
+      name="name"
+      value={$item.name}
+      class="item-name"
+      placeholder="Name"
+      on:change={({ target }) =>
+        updateDocumentDataFromField($item, target.name, target.value)}
     />
-    {#if $item.system.broken}
-        <i
-            class="broken-item-icon fa-solid fa-heart-crack"
-            data-tooltip="A5E.BrokenItem"
-            data-tooltip-direction="DOWN"
+    {#if hasType.includes($item.type)}
+      <div class="prerequisites">
+        <label class="prerequisite-label" for={`${appId}-prerequisites`}
+          >Prerequisites:
+        </label>
+        <input
+          id={`${appId}-prerequisites`}
+          type="text"
+          name="system.prerequisite"
+          value={$item.system.prerequisite || null}
```

Comment:
`value={$$item.system.prerequisite || ""}` Should leave it as an empty string.

Reply:
No need to even go that far. If an empty string correctly shows the placeholder text, then we can just do:
```js
value={$item.system.prerequisites}
```
That test is there to catch empty strings and replace them with `null`, presumably to deactivate the value field. Josh was saying this wasn't working with an empty string, but I wasn't around to actually test this. It could have been a consequence of something else.

Reply:
Yeah, just for context - this was leftover from previous iterations where the label was not correctly working.